components/ListComponents/index.js

```
@@ -78,9 +78,8 @@ export function ListNote() {
   return (
     <div className={`${css.note} ${utils.colXs12}`}>
       <p>
-        <strong>Note:</strong> You won&apos;t see lists created in another
-        browser here. To view those lists, open the browser you used when
-        creating them.
+        <strong>Note:</strong> Your lists are stored locally in your browser.
+        You won't see lists created in another browser here. If you want to save a list long-term, you may wish to use the "download list" feature to export your lists as CSV files.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)